bin/lib/correlations.js

```diff
@@ -4,9 +4,16 @@ const fetch = require('node-fetch');
 const CORRELATION_SERVICE_HOST = process.env.CORRELATION_SERVICE_HOST;
 const CORRELATIONS_SERVICE_TOKEN = process.env.CORRELATIONS_SERVICE_TOKEN;
 
-function getAllOfTheIslandsInTheCorrelationsService(){
+const REQUEST_HEADERS = {
+	method: 'GET',
+	headers: {
+		'Content-Type': 'application/json',
```

Comment:
Shouldn't this be `Accepts`? We're not passing a body in the request, so what are we declaring as JSON?